README.md

```diff
@@ -11,19 +11,15 @@
 
 These steps are to be completed by the author of the blog post.
 
-1. Create [an issue](https://github.com/OpenLiberty/blogs/issues/new) for the blog post. This is to help the editors track the progress of the post.
+1. Create an issue for the blog post. This is to help the editors track the progress of the post.
 
    * **All blog posts except release blog posts**
 
-     Create a normal issue. In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
+     Create [a normal issue](https://github.com/OpenLiberty/blogs/issues/new). In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
```

Comment:
```suggestion
     Create [a issue](https://github.com/OpenLiberty/blogs/issues/new). In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
```

Did not feel like we needed to describe what kind of issue to open, in this case "normal", since we now have a direct link to opening an issue.